atom/browser/osr/osr_render_widget_host_view_mac.mm

```diff
@@ -22,15 +22,15 @@ explicit MacHelper(OffScreenRenderWidgetHostView* view) : view_(view) {
   virtual ~MacHelper() {}
 
   // content::BrowserCompositorMacClient:
-  SkColor BrowserCompositorMacGetGutterColor(SkColor color) const override {
+  SkColor BrowserCompositorMacGetGutterColor() const override {
     // When making an element on the page fullscreen the element's background
     // may not match the page's, so use black as the gutter color to avoid
     // flashes of brighter colors during the transition.
     if (view_->render_widget_host()->delegate() &&
         view_->render_widget_host()->delegate()->IsFullscreenForCurrentTab()) {
       return SK_ColorBLACK;
     }
-    return color;
+    return SK_ColorTRANSPARENT;
```

Comment:
Are we sure this is what we want to do? :thinking: 


This code is lifted from upstream, which is  now using [this impl](https://cs.chromium.org/chromium/src/content/browser/renderer_host/render_widget_host_view_mac.mm?q=BrowserCompositorMacGetGutterColor&sq=package:chromium&dr=C&l=72). `last_frame_root_background_color_` is assigned in [RenderWidgetHostViewMac::OnRenderFrameMetadataChangedAfterActivation](https://cs.chromium.org/chromium/src/content/browser/renderer_host/render_widget_host_view_mac.mm?rcl=0589d25476de0308fa44775bd8ece4d81ae2375d&l=623)

Could we try [adding that](https://cs.chromium.org/chromium/src/content/browser/renderer_host/render_widget_host_view_mac.h?rcl=ea47556487f8916886130284de4c7a74f869eb7f&l=233) to our class as well to match upstream?

Reply:
We could match the behavior of the C66 version of that https://chromium.googlesource.com/chromium/src/+/66.0.3359.181/content/browser/renderer_host/render_widget_host_view_mac.mm